packages/ts-migrate/bin/ts-migrate-full.sh

```diff
@@ -76,17 +77,17 @@ maybe_commit -m "[ts-migrate][$folder_name] Init tsconfig.json file" -m 'Co-auth
 echo "
 [Step $((step_i++)) of ${step_count}] Renaming files from JS/JSX to TS/TSX and updating project.json\...
 "
-$cli rename $frontend_folder
+$cli rename $frontend_folder $additional_args
 
 maybe_commit -m "[ts-migrate][$folder_name] Rename files from JS/JSX to TS/TSX" -m 'Co-authored-by: ts-migrate <>'
 
 echo "
 [Step $((step_i++)) of ${step_count}] Fixing TypeScript errors...
 "
-$cli migrate $frontend_folder
+$cli migrate $frontend_folder $additional_args
 
 if [ "$should_remove_eslintrc" = "true" ]; then
-  rm -f $frontend_folder/.eslintrc.*
+  rm -f $frontend_folder/.eslintrc
```

Comment:
why did you change `.eslintrc.*` to `.eslintrc`? (for the context, #26)

Reply:
Good question. We have an .eslintrc.js that the script wasn't detecting (my guess is that I have different glob settings configured somewhere). This meant that an additional .eslintrc would get created, and then both of them would be deleted. 

I updated the check on line 70 to use stat, which is allegedly more portable. In addition, since this script only creates .eslintrc, I updated the deletion step here to only delete that file specifically. The update to the deletion step isn't necessary, but the changes on line 70 keep our .eslintrc.js from being deleted.

Reply:
great, thanks!